scripts/startNetwork.sh

```diff
@@ -6,15 +6,19 @@ set -e
 TEST_MODE=""
 while getopts 't' flag; do
   case "${flag}" in
-    t) TEST_MODE="-t" ;;
-    ?) printf 'Invalid flag!' ;;
+  t) TEST_MODE="-t" ;;
+  ?) printf 'Invalid flag!' ;;
   esac
 done
 
 # Set environment variables
 source ./scripts/env.sh
 source ./scripts/util.sh
 
+if [[ $TEST_MODE == "-t" ]]; then
+  ./scripts/startNetwork/setupLocalNodeIP.sh
```

Comment:
```suggestion
  ./startNetwork/setupLocalNodeIP.sh
```

are we not already in path ./scripts?

Reply:
We use absolute paths to avoid path problems. The scripts are initially called from the root folder :)